sycl/doc/extensions/accessor_properties/SYCL_ONEAPI_accessor_properties.asciidoc

```diff
@@ -72,10 +72,10 @@ Here are some examples of legal and illegal conversions:
 
 ```c++
 accessor A2(B, h, accessor_property_list{no_alias});
-accessor A3(B, h, accessor_property_list{no_alias, noinit});
+accessor A3(B, h, accessor_property_list{no_alias, no_init});
```

Comment:
Probably we should not update the extension specification. Right?

We can add a note that this extensions was added to SYCL 2020 specification with modifications.

Reply:
I suggest we remove `noinit`/`no_init` from this spec to avoid such maintenance.

Reply:
I think that @jessicadavies-intel is still working on this extension.  The references to `no_init` are there to illustrate the difference between runtime properties and compile-time properties, so I do not think we should remove them.

Reply:
The file currently in the PR is using the `no_init` spelling; while  `noinit` is the spelling for the extension. Shall I update this asciidoc to the `noinit` spelling?

Reply:
Based on the latest comment from @gmlueck, I think `no_init` spelling is preferred.

Reply:
Yes, `no_init` spelling is preferred.